doc/user_guide/system_specific_setup.rst

```diff
@@ -1,29 +1,21 @@
 .. _system_specific_setup:
 
-System-specific set-up
-======================
-This chapter is split into two sections: :ref:`setup_user`
-describes the setup for a user of PSyclone. It includes all steps necessary
-to be able to use PSyclone. And while you could obviously do
-some development, none of the required tools for testing or
-documentation creation will be installed.
-
-:ref:`dev_setup` describes the additional installation of
-all required tools to run tests and create documentation.
-
-Both sections have detailed instructions for Ubuntu 16.04.2 and 
-OpenSUSE 42.2 - if you are working with a different Linux
-distribution some adjustments will be necessary.
+System-specific Set-up for Users
+================================
 
-.. _setup_user:
+This chapter describes the setup for a user of PSyclone. It includes
+all steps necessary to be able to use PSyclone. And while you could
+obviously do some development, none of the required tools for testing
+or documentation creation will be installed.
 
-User set-up
------------
+The :ref:`system_specific_dev_setup` in the Developers' Guide
```

Comment:
Should this be `dev_guide:system_specific_dev_setup`?

Reply:
No, it doesn't have to be. However, it's probably better if it is (clearer that it's an external link) so I've changed it.